README.md

```diff
@@ -11,79 +11,78 @@
     # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
 
-Basic GeoIPS Plugin Template 
-=============================
+GeoIPS Test Data Repository Template
+====================================
 
-This template repository contains everything necessary to create a fully compatible GeoIPS Plugin Package.
-Each file within this repository contains appropriate modification instructions.
+This template repository contains everything necessary to create a test data repository for use with
+the GeoIPS processing infrastructure.
 
 Follow the 
 [step by step instructions](https://github.com/NRLMMD-GEOIPS/template_basic_plugin/blob/dev/docs/template_instructions.rst)
```

Comment:
Should use relative links in README.md so that the links always point to a file in the current branch.

```suggestion
[step by step instructions](./docs/template_instructions.rst)
```